recipe/vina.patch

```diff
@@ -101,19 +103,89 @@ index cf9d426..51609bd 100644
  clean:
  	rm -f *.o
 diff --git a/build/python/setup.py b/build/python/setup.py
-index 938732e..28b3d98 100644
+index b182fda..81e3347 100644
 --- a/build/python/setup.py
 +++ b/build/python/setup.py
-@@ -297,8 +297,6 @@ class CustomBuildExt(build_ext):
-         
+@@ -45,7 +45,7 @@ def in_conda():
+ 
+ def find_version():
+     """Extract the current version of AutoDock Vina.
+-    
++
+     The version will be obtained from (in priority order):
+     1. version.py (file created only when using GitHub Actions)
+     2. git describe
+@@ -56,7 +56,7 @@ def find_version():
+     if os.path.isfile(version_file):
+         with open(version_file) as f:
+             version = f.read().strip()
+-        
++
+         print('Version found: %s (from version.py)' % version)
+         return version
+ 
+@@ -72,8 +72,8 @@ def find_version():
+         return version
+     except:
+         pass
+-    
+-    init_file = os.path.join(base_dir, 'vina', '__init__.py') 
++
++    init_file = os.path.join(base_dir, 'vina', '__init__.py')
+     with open(init_file) as f:
+         for line in f:
+             version_match = re.match(r'^__version__ = "(.+?)"$', line)
+@@ -83,7 +83,7 @@ def find_version():
+ 
+                 print('Version found %s (from __init__.py)' % version)
+                 return version
+-    
++
+     raise RuntimeError('Could not find version string for AutoDock Vina.')
+ 
+ 
+@@ -149,10 +149,10 @@ def locate_ob():
+ def locate_boost():
+     """Try to locate boost."""
+     if in_conda():
+-        data_pathname = os.environ["CONDA_PREFIX"]
++        data_pathname = os.environ["PREFIX"]
```

Comment:
That shouldn't be necessary. Also, if `locate_boost` is used anywhere but during `setup.py`, it will give the wrong result (`PREFIX` is populated only during the build; `CONDA_PREFIX` also in the final environment).

Reply:
Boost is not found without it. I will put it back.

Reply:
Let me know if you have another idea - but using PREFIX seems to work.

Reply:
It's OK, it seems to only happen at build time.